src/rules/action-prop-name.ts

```diff
@@ -0,0 +1,50 @@
+import { ESLintUtils } from '@typescript-eslint/experimental-utils'
+import type { TSESTree } from '@typescript-eslint/experimental-utils'
+
+import { actionCreatorPropsComputed, docsUrl } from '../utils'
+
+export const ruleName = 'action-prop-name'
+
+export const actionPropName = 'actionPropName'
+export const actionPropNameSuggest = 'actionPropNameSuggest'
+export type MessageIds = typeof actionPropName | typeof actionPropNameSuggest
+
+type Options = []
+
+export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
```

Comment:
Let me know what do you think about the rule name and its description/failures.

Reply:
What do you think about prefer-inline-action-props ?

Reply:
Done. What about the description and failure messages?

Reply:
I think it's good.  👍
We'll provide more info in the docs folder soonish